src/screens/SettingsModal.js

```diff
@@ -60,37 +61,37 @@ const SettingsPages = {
   backup: {
     component: View,
     key: 'BackupSection',
-    title: 'Backup',
+    title: () => lang.t('settings.backup'),
   },
   currency: {
     component: CurrencySection,
     key: 'CurrencySection',
-    title: 'Currency',
+    title: () => lang.t('settings.currency'),
   },
   default: {
     component: null,
     key: 'SettingsSection',
-    title: 'Settings',
+    title: () => lang.t('settings.label'),
   },
   dev: {
     component: IS_DEV ? DevSection : null,
     key: 'DevSection',
-    title: 'Dev',
+    title: () => lang.t('settings.developer'),
   },
   language: {
     component: LanguageSection,
     key: 'LanguageSection',
-    title: 'Language',
+    title: () => lang.t('settings.language'),
   },
   network: {
     component: NetworkSection,
     key: 'NetworkSection',
-    title: 'Network',
+    title: () => lang.t('settings.network'),
   },
   privacy: {
     component: PrivacySection,
     key: 'PrivacySection',
-    title: 'Privacy',
+    title: () => 'Privacy',
```

Comment:
no lang.t for `Privacy` ?

Reply:
Interesting, I'm not sure why that didn't get included. I'll make sure to add it to the next round of translations!